config/manager/manager.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         runAsNonRoot: true
       containers:
       - command:
-        - /manager
+        - /usr/local/bin/manager
```

Comment:
Since the entrypoint is set, you can skip setting the `command` in the pod spec.

Reply:
Good point, I removed the `command` and left only the `args`.